src/main/java/org/skife/config/Default.java

```diff
@@ -10,4 +10,5 @@
 public @interface Default
 {
     String value();
+    String[] for_values() default {};
```

Comment:
I like `snake_case`, but I *think* `camelCase` is still idiomatic naming for attribute values. Any reason to maintain snake_case here?